content/en/blog/2021-06-28-bug-scrub-announce.md

```diff
@@ -0,0 +1,80 @@
+---
+author: kingdonb
+date: 2021-06-28 08:30:00+00:00
+title: Flux Bug Scrub Announce
+description: We are starting the Flux Bug Scrub initiative. A great way to help out in the community and get started by learning more about Flux and interacting with user issues!
+url: /blog/2021/06/flux-bug-scrub-announce/
+---
+
+## What is the Flux Bug Scrub
+
+In the past months we have had many folks reach out to us and ask where
+they could get involved, make their first steps in the Flux community
+and give back in a way.
+
+We started looking around and stole a concept from SIG Node (who also
+stole it from SIG CLI) over in the Kubernetes community. Because for us
```

Comment:
Let's use friendlier words not `stole`..

Reply:
:+1: